CreateInstBlobTrigger/institution_docs.py

```diff
@@ -0,0 +1,138 @@
+"""
+This module extracts institution information from the HESA
+XML dataset and writes it, in JSON format, to Cosmos DB.
+
+Currently, if expected data is missing, we let the exception
+bubble up.
+"""
+import datetime
+import inspect
+import logging
+import os
+import sys
+import xml.etree.ElementTree as ET
+
+import xmltodict
+
+# TODO investigate setting PATH in Azure so can remove this
+CURRENTDIR = os.path.dirname(
+    os.path.abspath(inspect.getfile(inspect.currentframe()))
+)
+PARENTDIR = os.path.dirname(CURRENTDIR)
+sys.path.insert(0, CURRENTDIR)
+sys.path.insert(0, PARENTDIR)
+
+from EtlPipelineBlobTrigger.course_docs import get_code_label_entry
+from SharedCode.utils import (
+    get_collection_link,
+    get_cosmos_client,
+    get_ukrlp_lookups,
+    get_uuid,
+)
+import course_lookup_tables as lookup
+
+
+class InstitutionDocs:
+    def __init__(self):
+        self.ukrlp_lookups = get_ukrlp_lookups()
+
+    def get_course(self, raw_course_data):
+        # TODO complete and call this
```

Comment:
Is this comment needed?

Reply:
My understanding is we may add a list of courses for the institution at some stage. I'd done a bit of work on this so this is just a place holder really for if/when we revisit it to add courses.